.nuspec/Microsoft.Maui.Controls.SingleProject.targets

```diff
@@ -49,23 +49,20 @@
 
   <!-- Removals -->
   <ItemGroup Condition=" '$(EnableDefaultItems)' == 'true' and '$(SingleProject)' == 'true' ">
+    <Compile Remove="$(PlatformsProjectFolder)**/*$(DefaultLanguageSourceExtension)" />
     <!-- Explicitly include the platform code -->
-    <_MauiCompileToAdd
+    <Compile
         Condition=" '$(TargetPlatformIdentifier)' == 'android' and '$(AndroidProjectFolder)' != '' "
         Include="$(AndroidProjectFolder)**/*$(DefaultLanguageSourceExtension)" />
-    <_MauiCompileToAdd
+    <Compile
         Condition=" '$(TargetPlatformIdentifier)' == 'ios' and '$(iOSProjectFolder)' != '' "
         Include="$(iOSProjectFolder)**/*$(DefaultLanguageSourceExtension)" />
-    <_MauiCompileToAdd
+    <Compile
         Condition=" '$(TargetPlatformIdentifier)' == 'maccatalyst' and '$(MacCatalystProjectFolder)' != '' "
         Include="$(MacCatalystProjectFolder)**/*$(DefaultLanguageSourceExtension)" />
-    <_MauiCompileToAdd
+    <Compile
         Condition=" '$(TargetPlatformIdentifier)' == 'windows' and '$(WindowsProjectFolder)' != '' "
         Include="$(WindowsProjectFolder)**/*$(DefaultLanguageSourceExtension)" />
-    <None Include="$(PlatformsProjectFolder)**/*$(DefaultLanguageSourceExtension)" Exclude="@(_MauiCompileToAdd)" />
```

Comment:
We didn't need this, tested:

![image](https://user-images.githubusercontent.com/840039/148838785-87b0570b-4a88-4d1f-bbbc-fad21141fae7.png)

Shows up in the IDE, no change to `.csproj`:

![image](https://user-images.githubusercontent.com/840039/148839069-f829fbf8-6fc0-4247-8580-aef2bebe02ff.png)

The file already is added to `@(None)`:

![image](https://user-images.githubusercontent.com/840039/148839144-b5535a7b-d2af-41db-b6ac-65ffdf938d10.png)

Reply:
I'm guessing since we added the .cs files to none this was added as well to balance that? Seems reasonable this isn't needed.